safechef/safechef.jsonld

```diff
@@ -0,0 +1,23 @@
+  "certificateNo": "8460c404-c0a5-4653-aeae-44d03a1a7334",
+  "examDate": "2011-11-11",
+  "testCode": "df52b514-e13a-42b7-8030-3f928a27d167"
```

Comment:
I'm not sure of the exact modeling for a certificate like this, but these certificate properties should be somewhere under `credentialSubject`.  There may be best practices to align with in VC-EDU or OpenBadges.  I'm not sure what the latest are though.

Food for thought:
https://w3c-ccg.github.io/vc-ed-models/#approach-2-open-badge-as-verifiable-credentials
They refer to 
https://imsglobal.github.io/openbadges-specification/ob_v3p0.html
Which has some `Certificate` references to the quite complex looking CTDL:
https://credreg.net/ctdl/terms

So many schemas... hard to know what to do!

- Properties for certificate itself should be under a property under `credentialSubject`.
- `issuanceDate` should be added to VC.
- Add an `id` under `credentialSubject` for jane doe, if possible.